### ISSUE TITLE
About DETR Align

### ISSUE THREAD

**Issue by becauseofAI** [liming-ai/AlignDet#8]
How does DETR perform alignment operations? 
Can I find a detailed description in the paper? 
Where is the code?

**Comment by liming-ai** [comment]
DETR works the same as other methods, except for a cluster-based bipartite graph matching and cross-entropy loss.

You can refer to Section B.5. for the detailed description in the paper.

Please refer to [DETR pre-training config](https://github.com/liming-ai/AlignDet/blob/master/configs/selfsup/detr.py) for more pre-training details, we provide [an example](https://github.com/liming-ai/AlignDet#pre-training-and-fine-tuning-instructions) to pre-train and fine-tune Mask R-CNN. The DETR is similar to it, and you should replace the related config.

**Comment by liming-ai** [comment]
Close this issue due to no response for 2 days. Feel free to re-open this issue if you have more questions.
